Dockerfile

```diff
@@ -10,14 +12,14 @@ COPY go.sum go.sum
 RUN go mod download
 
 # Copy the go source
-COPY main.go main.go
+COPY cmd/main.go cmd/main.go
```

Comment:
These changes will not work without the PR merged first (as there's no change cmd pkg).

Reply:
for now testing (draft PR) will remove `cmd`